sanic/worker/loader.py

```diff
@@ -35,7 +39,9 @@ def __init__(
 
         if module_input:
             delimiter = ":" if ":" in module_input else "."
-            if module_input.count(delimiter):
+            if module_input.count(
+                delimiter
+            ) and not module_input.upper().startswith("C:\\"):
```

Comment:
Suggest `delimeter in module_input and ":\\" not in module_input` (more Pythonic, accounting also for other drive letter than C).

Possibly something different than that to check for paths because on Windows `d:/directory` (i.e. using forward slash, different drive) should be a valid path too.